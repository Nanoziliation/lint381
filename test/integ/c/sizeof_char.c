sizeof(char);
sizeof char;
sizeof(int);
char a;

sizeof char;
sizeof int;
