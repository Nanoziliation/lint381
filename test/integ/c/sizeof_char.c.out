sizeof_char.c:1:1: error: sizeof(char) should not be used as it is defined by the C standard to be equal to 1
sizeof(char);
^^^^^^^^^^^
sizeof_char.c:2:1: error: sizeof(char) should not be used as it is defined by the C standard to be equal to 1
sizeof char;
^^^^^^^^^^^
sizeof_char.c:6:1: error: sizeof(char) should not be used as it is defined by the C standard to be equal to 1
sizeof char;
^^^^^^^^^^^
